ydb/core/fq/libs/http_api_client/http_client.py

```diff
@@ -289,3 +292,9 @@ def get_openapi_spec(self) -> str:
 
     def compose_query_web_link(self, query_id) -> str:
         return self._compose_web_url(f"/folders/{self.config.project}/ide/queries/{query_id}")
+
+    @staticmethod
+    def result_set_to_dataframe(data):
+        import pandas as pd
```

Comment:
Let's move import to the top of the file. It would be very inconvenient to conditionally recieve import error =)